### ISSUE TITLE
Investigate and add bandaid fix for ReloadAll for Firefox Homepage

### ISSUE THREAD

**Issue by data-sync-user** [mozilla-mobile/firefox-ios#11332]
The idea for the bandaid fix is that we have a high-frequency crash that is happening since v102 as we made changes to the Firefox Home page which has a compositional layout. 

We’ll aim to find a bandaid fix is possible while TECH Team works in parallel to improve the overall ReloadAll State and Firefox Homepage structure. 

Another issue that is related to this reload all issue has been added below.

[https://bugzilla.mozilla.org/show_bug.cgi?id=1779505](https://bugzilla.mozilla.org/show_bug.cgi?id=1779505|smart-link) - This is the crash that happens in Firefox Home for when it reloads all sections and is yet to be fixed

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/FXIOS-4559)
